client/CServerHandler.cpp

```diff
@@ -924,7 +924,7 @@ void CServerHandler::threadHandleConnection()
 	try
 	{
 		sendClientConnecting();
-		while(c->connected)
+		while(c && c->connected)
```

Comment:
Does not it crash later?

Reply:
void CServerHandler::endGameplay calls connection a few times

Reply:
as I testes, was good. Will download build from PR later

Reply:
tested on build, everything is fine. Did you reproduce crash on PR build?